service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/create/GetGcpCloudContextStep.java

```diff
@@ -1,47 +0,0 @@
-package bio.terra.workspace.service.resource.controlled.flight.create;
-
-import bio.terra.stairway.FlightContext;
-import bio.terra.stairway.Step;
-import bio.terra.stairway.StepResult;
-import bio.terra.stairway.exception.RetryException;
-import bio.terra.workspace.service.iam.AuthenticatedUserRequest;
-import bio.terra.workspace.service.workspace.GcpCloudContextService;
-import bio.terra.workspace.service.workspace.flight.WorkspaceFlightMapKeys.ControlledResourceKeys;
-import bio.terra.workspace.service.workspace.model.GcpCloudContext;
-import java.util.UUID;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-public class GetGcpCloudContextStep implements Step {
```

Comment:
Unused step, this was replaced by `GetCloudContextStep` which also handles Azure contexts